pkg/drivers/kic/oci/network_create.go

```diff
@@ -36,6 +37,12 @@ const firstSubnetAddr = "192.168.49.0"
 // big enough for a cluster of 254 nodes
 const defaultSubnetMask = 24
 
+// will be used if docker bridge config doesn't exist related issue #9528
+const defaultMTU = 1500
```

Comment:
As discussed offline, I agree with you that the MTU setting should be omitted if it can't be discovered.